src/sql-parser/src/parser.rs

```diff
@@ -2234,6 +2234,10 @@ impl<'a> Parser<'a> {
                         format!("No value parser for keyword {}", kw)
                     );
                 }
+                Token::Op(ref op) if op == "-" => match self.next_token() {
```

Comment:
@benesch how did we handle negative literals before this?

Reply:
I don't think there were negative literals before, just unary negation expressions. It's just that in this position we don't expect an expression, just a plain value